spec/models/plan_year_spec.rb

```diff
@@ -1980,7 +1980,7 @@ def p(obj)
     let(:plan_year_end_on) { Date.new(calendar_year + 1, 3, 31) }
```

Comment:
why do we need to change the saving order of these plna years?

Reply:
See line 1999, Selecting data after mocking.